sw/rust/ibex-demo-system-pac/.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+target = "riscv32imc-unknown-none-elf"
+
+[unstable]
+build-std = ["core"]
```

Comment:
Can we have this in the top-level `.cargo/config.toml` so that we can get rid of this config?

Reply:
Removed